src/pint/models/pulsar_binary.py

```diff
@@ -102,7 +102,9 @@ def __init__(self):
         )
         self.add_param(
             floatParameter(
-                name="A1", units=ls, description="Projected semi-major axis, a*sin(i)"
+                name="A1",
+                units=ls,
+                description="Projected semi-major axis of pulsar orbit",
```

Comment:
Why not leave in the specific definition?

Reply:
reverted.